### ISSUE TITLE
Dotnet CLI Tools - The WiX Toolset v3 build tools must be installed to build

### ISSUE THREAD

**Issue by stoned72** [wixtoolset/issues#5607]
Please provide answers to the following questions to help us narrow down, reproduce, and fix the problem. Fill out one section and delete the others.

Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> 3.11.0.1507

* Which version of Visual Studio are you building with (if any)?

> VS2017 - Version 15.1 (26403.3)   /  .NET CLI Tools 1.0.3

* Which version of .NET are you building with?

> 4.6.2 / .NET Core

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> N/A

* Describe the problem and the steps to reproduce it.

> I have added a Wix Setup project to my solution and this builds fine within the VS2017 IDE and msbuild on the command line. However, as the solution contains an ASP.NET Core project, I also need to build via the .NET CLI Tooling on build server using this command...

dotnet build mysolution.sln

However, when build via dotnet.exe I get the following error...

...SetupProject.wixproj(27,5): error : The WiX Toolset v3 build tools must be installed to build this project. To download the WiX Toolset, see http://wixtoolset.org/releases/

* Describe the behavior you expected and how it differed from the actual behavior.

> Please confirm what is required in order to build the wix project using dotnet.exe?

**Comment by robmen** [comment]
First, you need to upgrade to final release of the WIX Toolset v3.11.

**Comment by stoned72** [comment]
I've installed version 3.11.1701.0, created a new wix v3 Setup Project via the VS2017 IDE. This again builds ok in Visual Studio but fails with the same error when building with the DotNet CLI as above.

**Comment by Milosdjordjevic84** [comment]
Any news about this?

**Comment by robmen** [comment]
First, you don't need to use dotnet.exe to build ASP.NET Core projects. You can use MSBuild and in doing so, it should build all projects. This will be the solution for WiX v3..x.

In WiX v4, this _should_ start working as we modernize the tooling. But there may still be cases that don't work since we've found dotnet.exe can't build .vcxprojs correctly in all cases either.

**Comment by eddex** [comment]
I tried to build my WiX project from the CLI and run into the same issue.

I installed the latest version of the WiX Toolset `v3.11.2.4516`.

I tried to build it with the following command:
```bash
msbuild -t:restore MyProject.sln /p:Configuration=Debug /p:Platform="x86"
```
But I get the following warning: `warning NU1503: Skipping restore for project 'C:\pathtomyproject\MyProject.wixproj'. The project file may be invalid or missing targets required for restore.`

If I build it with Visual Studio, everything works fine.
